src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -83,7 +86,7 @@ $params = New-Object PSObject -Property @{
 $params | Set-AzVmDiskEncryptionExtension
 ```
 
-This example demonstrates sending parameters using pipelined input to enable encryption without specifying AD credentials.  
+This example enables encryption on a Windows VM by sending parameters using pipelined input without specifying AD credentials.  
```

Comment:
I think this would work on Linux too?   If so "on a Windows VM" could be omitted.